src/repository/finding.py

```diff
@@ -33,6 +34,25 @@ def get_findings_by_task_id(
         )
 
         return findings
+    
+    def get_findings_by_task_id_and_severity(
+        self, task_id: int, severity: SeverityFilter, pagination: PaginationInput
+    ) -> list[db_models.Finding]:
+        findings = (
+            self.session.query(db_models.Finding)
+            .join(db_models.RecommendationTask)
+            .where(
+                db_models.RecommendationTask.status == db_models.TaskStatus.COMPLETED,
+                (db_models.RecommendationTask.id == task_id),
+                db_models.Finding.severity >= severity.minValue,
```

Comment:
@ishworgiri1999 Do you understand why the comparison doesn't work? When I do this, my findings are always empty😟

Reply:
@niklas1531  I never saved severity, and priority to db. hopefully, it should work now.

Reply:
Unfortunately not